wp-content/themes/core/assets/pcss/global/reset.pcss

```diff
@@ -0,0 +1,71 @@
+/* -------------------------------------------------------------------------
+ *
+ * Global "Resets" & Light Normalize
+ *
+ * ------------------------------------------------------------------------- */
+
+/**
+ * Use a more-intuitive box-sizing model.
+ */
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+/**
+ * Prevent adjustments of font size after orientation changes in iOS.
+ */
+html {
+	text-size-adjust: 100%;
+}
+
+/**
+ * 1. Improve text rendering.
+ * 2. Prevent horizontal scroll.
+ */
+body {
+	-webkit-font-smoothing: antialiased;
+	overflow-x: hidden;
+}
+
+/**
+ * Avoid text overflows.
+ */
+p,
+:--heading {
+	overflow-wrap: break-word;
+	hyphens: auto;
+}
```

Comment:
I don't think you _need_ both of these here. From the docs on [overflow-wrap](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) and [hyphens](https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens), it seems like you can just use `hyphens`, but `overflow-wrap` would be a good fallback if `hyphens` isn't supported for some reason.

Reply:
Not all languages support hypens, which is why I'm using `overflow-wrap`. The `overflow-wrap: break-word` rule also looks odd in English when words just break without hypens, so I opted to use both.

There's a big more info [here](https://www.joshwcomeau.com/css/custom-css-reset/#eight-word-wrapping-9) on both rules. Appreciate you poking at this though, it's always interesting reading up on the technicalities!